src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/Offender.java

```diff
@@ -166,6 +166,11 @@ public class Offender extends AuditableEntity {
     @Column(name = "LAST_NAME_ALPHA_KEY")
     private String lastNameAlphaKey;
 
+    @OneToMany(mappedBy = "offender", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
not really understanding this mapping.  Does that mean that each offender record can have addresses - is there no mapping by the root offender?